.pre-commit-config.yaml

```diff
@@ -61,6 +61,7 @@ repos:
             "--frozen",
             "--no-hashes",
             "--no-emit-project",
+            "--no-default-groups",
```

Comment:
I don't know if this is right, not that I care too much about requirements.txt :)